docs/UserGuide.md

```diff
@@ -1,134 +1,301 @@
+Important: The square brackets are compulsory when adding an event.
+
+####1.2 Add a deadline
+
+>Format: **Add** &lt;deadline description&gt;; &lt;date(DDMMYY)&gt;; &lt;end time&gt;
+
+Example: Add complete report; 120916; 1900
+
+>Important: The date and end time are compulsory when adding an deadline
+
+####1.3 Add a to-do
+
+>Format: **Add** &lt;to-do description&gt;
+
+Example: Add go swimming
+
+>Note: To do tasks do not have a date or time.
+
+>***Note: To add a tag, simply add a # followed by the tag name at the back of the command 
+To add a priority, simply add a ! at the back of the command.***
+
+####1.4 Add a tag / multiple tags
+
+>Format: **Add** &lt;index&gt; #tag1 #tag2
+
+Example: Add E1 #win #victor #finally  
+
+###2 Edit a task
+
+The multiple edit task functions allow you to update specific task details.
+
+####2.1 Edit all task details in one go
+
+>Format: **Edit** &lt;index&gt; [&lt;event description&gt;; &lt;date&gt;; &lt;start time&gt;; &lt;end time&gt;; &lt;tags&gt;; !] 
+
+>Format: **Edit** &lt;index&gt; &lt;deadline description&gt;; &lt;date&gt;; &lt;end time&gt;; &lt;tags&gt; 
+ 
+>Format: **Edit** &lt;index&gt; &lt;to-do description&gt;; !
+
+To add certain task details like start time, simply add their corresponding sections in the command format.
+
+To remove certain task details like end time, simply leave their corresponding sections in the command format blank.
+
+>Note: Adding a start time and end time to a to-do will automatically convert your to-do into and event.
+
+>Note: Adding a end time to a to-do will automatically convert your to-do into a deadline.
+
+>Note: Adding a start time to a deadline will automatically convert a deadline to an event.
+
+>Note: Add a date to a to-do will automatically convert your to-do into a deadline with a default end time of 2359 on that date.
+
+>Note: Removing date from events and deadlines will automatically convert them into a to-do with no date or time.
+
+>Note: Removing start time from events will automatically convert them into deadlines.
+
+>Note: Removing end time from deadlines will automatically set the end time of the deadline to 2359. 
+ 
+####2.2 Edit task description
+
+>Format: **Edit** &lt;index&gt; **des** &lt;description&gt;
+
+Example: Edit T1 des eat apple
+
+####2.3 Edit task date
+
+>Format: **Edit** &lt;index&gt; **date** &lt;date&gt;
+
+Example: Edit E1 date 12/09/16
+
+>Note: If you add a date to a to-do, this will automatically convert your to-do into a deadline with a default end time of 2359 on that date.
+
+####2.4 Edit task start time 
+
+>Format: **Edit** &lt;index&gt; **start** &lt;start time&gt;
+
+Example: Edit E1 start 7am
+
+>Note: Adding a start time to a to-do will automatically convert your to-do into an event with a default day of today and start time as entered. 
+
+####2.5 Edit task end time 
+
+>Format: **Edit** &lt;index&gt; **end** &lt;end time&gt;
+
+Example: Edit E1 end 1900
```

Comment:
If any of the keywords like end show up in the description make sure to handle it.